### DISCUSSION TITLE
Metric for time

### DISCUSSION THREAD

**Discussion by Linardos** [FETS-AI/Challenge#108] [Q&A] [answered]
I was curious to find that the time in the resulting dataframe of an experiment is lower than the amount of time the experiment took (The experiment took 2+days, but the displayed seconds translate to 10hours). Is there any reason simulated_time (as described in README) would show less than the actual time? I thought simulated time would add to it as it accounts for download and upload communication cost that is not really present since all data is gathered in one machine.
![image](https://user-images.githubusercontent.com/26694607/126168466-57d8fa06-484b-4e67-be69-6a471ade3f9b.png)

**Comment by sarthakpati** [comment, marked as answer]
Hi Linardos,

Thank you for your interest in the FeTS Challenge!

Since we will be running the validation (i.e., re-training of all methods) on a common hardware setup, you will not need to worry about whether the communication cost is normalized or not, because all participants will be on a level playing field.

It is expected that there will be a difference in the simulated time and wall clock time, so no worries!

Cheers,
Sarthak
